packages/coding-standard/src/Rules/ForbiddenFuncCallRule.php

```diff
@@ -51,12 +52,24 @@ public function process(Node $node, Scope $scope): array
         }
 
         $funcName = $node->name->toString();
+        foreach ($this->forbiddenFunctions as $forbiddenFunction) {
+            $errorMessage = [sprintf(self::ERROR_MESSAGE, $forbiddenFunction)];
 
-        if (! in_array($funcName, $this->forbiddenFunctions, true)) {
-            return [];
+            if ($funcName === $forbiddenFunction) {
+                return $errorMessage;
+            }
+
+            $isEndWithMask = Strings::endsWith($forbiddenFunction, '*');
+            $isStartWithFunctionForbidden = Strings::startsWith(
```

Comment:
It's better to cover with fnmatch. As no `*_some` or `func_*_all` would not work now.

You can re-use this service: https://github.com/symplify/symplify/blob/7719288efc47f38cad3d4bc18992749bf3fbd682/packages/package-builder/src/Matcher/ArrayStringAndFnMatcher.php#L12

Reply:
implemented.